pyproject.toml

```diff
@@ -1,7 +1,67 @@
+[build-system]
+requires = ["hatchling"]
```

Comment:
Could you tell me a little more about hatchling? Why did you choose this over other options such as `setuptools`?

Reply:
It is the build backend used in the [tutorial](https://packaging.python.org/en/latest/tutorials/packaging-projects/) I followed on python.org for packaging. I like their [website](https://hatch.pypa.io/latest/), it was easy to browse and find information quickly. It is well supported and extensively used in the Python community (see list of users [here](https://hatch.pypa.io/latest/community/users/)).

That said, the OGE project being relatively small and its architecture is pretty simple, any build backend will perform well. We can explore other backends if you want. We used `setuptools` for packaging our projects at Breakthrough Energy and it worked just fine.